android/app/src/main/AndroidManifest.xml

```diff
@@ -12,26 +12,30 @@
         android:targetSdkVersion="22" />
 
     <application
-      android:name=".MainApplication"
-      android:allowBackup="true"
-      android:label="@string/app_name"
-      android:icon="@mipmap/ic_launcher"
-      android:theme="@style/AppTheme">
-      <activity
-        android:name=".MainActivity"
+        android:name=".MainApplication"
+        android:allowBackup="true"
         android:label="@string/app_name"
-        android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-        android:windowSoftInputMode="adjustResize">
```

Comment:
Removing this has introduced layout issue.
When keyboard pops up navbar is removed from the screen.
<img width="449" alt="screen shot 2017-06-15 at 10 10 49 am" src="https://user-images.githubusercontent.com/18511177/27165648-fd64c432-51b2-11e7-8c47-628cd622ad25.png">

Reply:
Would you fix the code to have only the needed lines produce a diff (no indentation where none is needed)? It is hard to determine what was added.